DESCRIPTION

```diff
@@ -24,7 +24,7 @@ Imports:
     dataone,
     datapack,
     digest,
-    EML (== 1.0.1),
```

Comment:
Since EML 2 is on CRAN, can we just build off that version instead of GitHub?

Reply:
this is weird I wonder if I messed up merging in some upstream changes from master

Reply:
Did you have a merge conflict you had to manually resolve on `DESCRIPTION`? Pretty easy to do.

Reply:
yeah that is a good idea, I think I started this pre-CRAN and never updated

Reply:
Seems possible it was in https://github.com/NCEAS/arcticdatautils/pull/135/commits/20def312ba922122c5bf2d478761cef78c611b1e that this happened.